src/api/script/api_test_in_spec.sh

```diff
@@ -63,15 +63,12 @@ EOF
 /usr/sbin/memcached $MEMCACHED_USER -l 127.0.0.1 -d -P $MEMCACHED_PID_FILE || exit 1
 
 # migration test
```

Comment:
Let me try to answer some parts of it. When the db:structure:dump is called (be it directly or at the end of running db:{migrate,rollback}), it creates a structure.sql of the current db.

Important here is that it's not just any mysql DB, but exactly that one. So if we diff the structure.sql from git with the one from the 2.5 dump we see differences in:

- the db version
- db name
- indexing type that was used back then and now
- maybe more

That's why there is some [additional cleanup / sanitization](https://github.com/openSUSE/open-build-service/pull/3348/commits/21e7a61fe21d05dcbfc6f9db808ec22e911cf958#diff-3f4b8ede6f6e7c07a3b6d127b6d44b0cR6) needed.